package learning;import java.io.IOException;public class ExceptionTest1 {    public static void main(String[] args) {        ExceptionTest1 ex = new ExceptionTest1();        try {            ex.test(0);        } catch (IOException e) {            e.printStackTrace();        }    }    public void test(int i) throws IOException {        if (i == 0) {            throw new IOException("this is a test RuntimeException");        }        System.out.println("111");    }}